config/env/production/database.ts

```diff
@@ -0,0 +1,17 @@
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+export default ({ env }) => ({
+  connection: {
+    client: "postgres",
+    connection: {
+      host: env("DATABASE_HOST", process.env.DATABASE_HOST),
+      port: env.int("DATABASE_PORT", process.env.DATABASE_PORT),
+      database: env("DATABASE_NAME", process.env.DATABASE_NAME),
+      user: env("DATABASE_USERNAME", process.env.DATABASE_USERNAME),
+      password: env("DATABASE_PASSWORD", process.env.DATABASE_PASSWORD),
+      ssl: env.bool("DATABASE_SSL", process.env.DATABASE_SSL),
+    },
+  },
+});
```

Comment:
why the duplication of files?

Reply:
https://docs.strapi.io/dev-docs/configurations/environment#environment-configurations

Environment configs are defined in a `./config/env/{environment}/{filename}` fashion, since we have development and production, we need two files here.

Reply:
Would it be possible to have a utils fn imported in both so we don't have 2 identical files?

Reply:
Upon further research I've noticed we can also leave the file at `/env` level and strapi is smart enough to pull that in apparently. Updating this.